firefox_puppeteer/tests/test_toolbars.py

```diff
@@ -2,8 +2,6 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
-import unittest
```

Comment:
It was only used for the @unittest.skip() decorator. Given that it has been removed we don't need it anymore in this file.